geonode_mapstore_client/client/js/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from "react";
+import uniq from 'lodash/uniq';
+import PropTypes from "prop-types";
+
+import Button from "@js/components/Button";
+import FaIcon from "@js/components/FaIcon";
+import useLocalStorage from "@js/hooks/useLocalStorage";
+import Message from "@mapstore/framework/components/I18N/Message";
+
+const Accordion = ({
+    title,
+    titleId,
+    identifier,
+    content
+}) => {
+    const [accordionsExpanded, setAccordionsExpanded] = useLocalStorage('accordionsExpanded', []);
+    const isExpanded = accordionsExpanded.includes(identifier);
+    const onClick = () => {
+        const expandedList = isExpanded
+            ? accordionsExpanded.filter(expanded => expanded !== identifier)
+            : uniq(accordionsExpanded.concat(identifier));
+        setAccordionsExpanded(expandedList);
+    };
+    return (
+        <div className={'gn-accordion'}>
```

Comment:
Accordion should be all clickable and not only the chevron.
We should add color for hover:
--gn-accordion-bg;
--gn-accordion-color;
--gn-accordion-hover-bg;
--gn-accordion-hover-color;